opencog/openpsi/modulator.scm

```diff
@@ -56,3 +62,16 @@
 "
     (psi-get-functionality tag-node "updater")
 )
+
+
+; =============================================================================
+; CREATE MODULATORS
+
```

Comment:
The things below do not belong in this directory, they belong in `ros-behavior-scripting` or something like that.

The point is that we need to be able to use openpsi for things other than Hanson robots, and so should not be putting HR-specific code for Eva/Sophia into this repo.   Other things could be, for example, non-humanoid actors in minecraft, or voice-controlled fork-lift trucks -- all of which will need to use openpsi, but won't typically have concepts of "arousal" or positive/negative sentiment, etc.